src/acceptance/_bootstrap.php

```diff
@@ -1,5 +1,5 @@
 <?php
 if (!defined('TIMEOUT'))
 {
-	define('TIMEOUT', 30);
+	define('TIMEOUT', 5);
```

Comment:
?? why do you need this?

Reply:
As discussed: imo 30s is way to long. It will work when you stop on first failure, but otherwise each failing system test using this value will take 30s.